[2024-10-29T23:23:54.341+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: hook.insert_data_task manual__2024-10-29T23:23:51.941839+00:00 [queued]>
[2024-10-29T23:23:54.351+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: hook.insert_data_task manual__2024-10-29T23:23:51.941839+00:00 [queued]>
[2024-10-29T23:23:54.351+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-10-29T23:23:54.351+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2024-10-29T23:23:54.351+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-10-29T23:23:54.363+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): insert_data_task> on 2024-10-29 23:23:51.941839+00:00
[2024-10-29T23:23:54.368+0000] {standard_task_runner.py:55} INFO - Started process 356 to run task
[2024-10-29T23:23:54.370+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'hook', 'insert_data_task', 'manual__2024-10-29T23:23:51.941839+00:00', '--job-id', '90', '--raw', '--subdir', 'DAGS_FOLDER/hooks.py', '--cfg-path', '/tmp/tmpv4jdi3ct']
[2024-10-29T23:23:54.371+0000] {standard_task_runner.py:83} INFO - Job 90: Subtask insert_data_task
[2024-10-29T23:23:54.419+0000] {task_command.py:388} INFO - Running <TaskInstance: hook.insert_data_task manual__2024-10-29T23:23:51.941839+00:00 [running]> on host 55e2175811f2
[2024-10-29T23:23:54.477+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=hook
AIRFLOW_CTX_TASK_ID=insert_data_task
AIRFLOW_CTX_EXECUTION_DATE=2024-10-29T23:23:51.941839+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-10-29T23:23:51.941839+00:00
[2024-10-29T23:23:54.484+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2024-10-29T23:23:54.488+0000] {sql.py:375} INFO - Running statement: insert into teste2 values(1);, parameters: None
[2024-10-29T23:23:54.489+0000] {sql.py:384} INFO - Rows affected: 1
[2024-10-29T23:23:54.489+0000] {python.py:177} INFO - Done. Returned value was: None
[2024-10-29T23:23:54.498+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=hook, task_id=insert_data_task, execution_date=20241029T232351, start_date=20241029T232354, end_date=20241029T232354
[2024-10-29T23:23:54.541+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2024-10-29T23:23:54.566+0000] {taskinstance.py:2578} INFO - 1 downstream tasks scheduled from follow-on schedule check
